### PR TITLE
Improve docker demo doc

### PR DESCRIPTION
**Pull request by neiljerram**

### CODE REVIEW

**Comment by neiljerram** [comment]
Hi Spike,

This PR has various changes to the docs that I made while acquainting myself with it.  Please could you take a look and let me know what you think?

Thanks - Neil

**Comment by tomdee** [comment]
This is pretty impossible to review at the moment (because of the changed line length?). Can you think of some way to make this reviewable?

File: README.md [hunk 1]
```diff
@@ -1,72 +1,111 @@
...
- --node-image=<DOCKER_IMAGE_NAME>    Docker image to use for
-                          Calico's per-node container
-                          [default: calico/node:v0.0.6]
-
-```
+# Calico on Docker
+
+Calico can provide networking in a Docker environment. Each container gets its
+own IP, there is no encapsulation and it can support massive scale. For more
+details see http://www.projectcalico.org/technical/.
```
Comment (alexwlchan) [inline README.md]: This link is broken. It's been fixed on master (see issue #44), so this version of the link should be changed before this branch is merged, or we'll re-break the link.

**Comment by alexwlchan** [comment]
I've made a commit to the awlc-demo-doc branch which has the changes to GettingStarted.md but without the line wrapping, which is hopefully easier to review.

Something weird is going on with the line endings in README.md that's causing GitHub's diffs to play up, so I haven't been unable to unwrap that.

**Comment by spikecurtis** [comment]
Manually merged to master e6d0e2f5d394c266d101eaead8a53a1c8061ba30 and 7cf9a52a923b407c50fa32e787a112f3f5d3fc95

**Pull request closed without merging**